salt/states/zabbix_host.py

```diff
@@ -143,7 +144,25 @@ def _interface_format(interfaces_data):
         if set(groups) != set(cur_hostgroups):
             update_hostgroups = True
 
-        hostinterfaces = __salt__['zabbix.hostinterface_get'](hostids=hostid)
+        hosttemplates = __salt__['zabbix.template_get'](hostids=hostid, **kwargs)
```

Comment:
The develop branch version of this fine includes another function, assign_templates. You may want to base this off of that functionality as it allows for specifying a template via a templateID or name.